Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml

```diff
@@ -65,58 +65,55 @@
     <constant name="FwdTracker_z2" value="2000*mm"/>
     <constant name="FwdTracker_zOffset" value="14000*mm"/>
```

Comment:
Hi Joschka, 

* There are several reasons, why the rMax changed in the forward region. I've realized these after the FCC-hh meeting, when several clever comments were made... First, I changed the modules sizes to be the same as in barrel region (the less different types of modules, the lower price). In addition, with such size and given pitch, we have 256 or 512 or 1024 read-out channels (dividable by 2), which is what one in reality wants. We can then easily estimate the number of needed chips, which affects the total cost. After recalculation of all positions to have a hermetic detector, one comes up with a different rMax. Second, Werner always wanted the detectors to end-up at r=155cm, with a new module size I'm closer to that value...
* If there is a space of 10-15cm for services above the helpFwd disc (so between the fwd disc and the calorimeter), this shouldn't be a problem.
* Sure, I will propagate them. Good point!